api/address.proto

```diff
@@ -9,10 +9,11 @@ message Address {
       TCP = 0;
     }
     Protocol protocol = 1;
-    // For listeners, bind to 0.0.0.0 or :: to support the use of prefix/suffix
-    // matching in FilterChainMatch. For clusters, an address may be either an
-    // IP or hostname to be resolved via DNS.
-    google.protobuf.StringValue address = 2;
+    // For listeners, an empty address implies a bind to 0.0.0.0 or ::. This
```

Comment:
Can you elaborate on a specific example here? My understanding of the iptables redirect comes from `use_original_dst` and is at the port level. Can you give a simpler Listener example showing how does this work with respect to the destination IP address and the FilterChain table?